### ISSUE TITLE
Show a graphic for the “Power User Curve” — also commonly called the activity histogram or the L30

### ISSUE THREAD

**Issue by atom-box** [piwik/piwik#19397]
A user has requested a widget could be added to visualize a common marketing histogram called the Power Curve, popularized by Facebook.

Power Curve or L30 is described in many marketing blogs.
Examples here and here
https://www.reforge.com/brief/understand-your-most-engaged-users-with-the-power-user-curve#iHS6ZR0ba7wb-ulnrDIBNA
https://andrewchen.com/power-user-curve/

The L30 ![](https://reforge-brevity-uploads-prod.s3.amazonaws.com/brief/uploads/post/image_path/88/emma1533762108332.png)

A description from the second link, above:

_the L30 is a histogram of users’ engagement by the total number of days they were active in a month, from 1 day out of the month to all 30 (or 28, or 31) days. While typically reflecting top-level activity like app opens or logins, it can be customized for whatever action you decide is important to measure for your product._

_The Power User Curve has a number of advantages over DAU/MAU:_

-     It shows if you have a hardcore, engaged segment that’s coming back every day.
-     It shows the variability among your users: some are slightly engaged, whereas others are power users. Contrast this with DAU/MAU: it’s a single number and so blurs this variance.
-     When mapped to cohorts, Power User Curves let you see if your engagement is getting better over time — which in turn helps assess product launches and performance of other feature changes.
-     Power User Curves can be shown for different user actions, not just app opens. This matters if the core activity that matters for your product is deeper in the funnel.

_In other words, while the DAU/MAU gives you a single number, the Power User Curve gives entrepreneurs several avenues of analysis to assess their product’s engagement to the most addicted users — in a single snapshot, over time, and also in relation to monetization_

**Comment by atom-box** [comment]
An existing widget is pretty close, just is not a histogram. And is not exactly a Power Curve
This can be seen at Matomo > Behavior > Engagement > "Visits by Days Since Last Visit" 

![image](https://user-images.githubusercontent.com/22874146/175825454-7de2422b-2c7e-4ece-be32-4d15367633be.png)
